doc/source/programs/gdal_raster_scale.rst

```diff
@@ -0,0 +1,103 @@
+.. _gdal_raster_scale_subcommand:
+
+================================================================================
+"gdal raster scale" sub-command
+================================================================================
+
+.. versionadded:: 3.11
+
+.. only:: html
+
+    Scale the values of the bands of a raster dataset
+
+.. Index:: gdal raster scale
+
+Synopsis
+--------
+
+.. program-output:: gdal raster scale --help-doc
+
+Description
+-----------
+
+:program:`gdal raster scale` can be used to rescale the input pixels values
+from the range :option:`--srcmin` to :option:`--srcmax` to the range
+:option:`--dstmin` to :option:`--dstmax`.
+It is also often necessary to reset the output datatype with the :option:`--ot` switch.
+If omitted the output range is 0 to 255.
+If omitted the input range is automatically computed from the source dataset.
+This may be a slow operation on a large source dataset, and if using it multiple times
+for several gdal_translate invocation, it might be beneficial to call
+``gdal raster info --stats {source_dataset}`` priorly to precompute statistics, for
+formats that support serializing statistics computations (GeoTIFF, VRT...)
+Note that the values specified for the ranges are only used to compute a scale and
+offset to apply to the input raster values. In particular, ``srcmin`` and
+``srcmax`` are not used to clip input values unless :option:`--exponent`
```

Comment:
I think the fact that `--exponent` triggers different behavior is an accident? That's the impression I had when documenting this previously. Can we decouple the decision to clip or not clip from the value of `--exponent` ?